benchmarks/trees/run.py

```diff
@@ -181,6 +181,12 @@ def parse_args():
         ),
     )
     parser.add_argument("-niters", default=5, type=int, help=("Number of iterations for each experiment"))
+    parser.add_argument(

```

Comment:
Is this actually needed? I think it's a bit redundant since we can we use something like `batch_size <0` to trigger this behavior. I don't have really strong opinions on this, if you think it is easier for having "no batching experiments".

Reply:
My intention is, with `batch_benchmark=true`, we want to benchmark on the batch size specified in `batch_size` argument (10000 by default), so `batch_size < 0` doesn't make sense unless you are saying we can use the whole dataset as one batch. I think it is better to able to do benchmark on different batch size (1, 100, 10000 etc).